### PR TITLE
Refactor pegasus markdown documentation

### PR DESCRIPTION
**Pull request by Hamms**
Specifically:

1. Move everything to a new location at `/editing`
2. Separate out the subset of documentation that's actually markdown-specific from everything that's not
3. Add redirects from all old urls to new ones.

Check it out at https://adhoc-refactor-markdown-documentation.cdn-code.org/editing

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Comment by dju90** [comment]
I believe the [wiki link](http://wiki.code.org/display/PROD/How+to+add+website+content+via+Dropbox+or+Gsheets) on the [Template Concepts](https://adhoc-refactor-markdown-documentation.cdn-code.org/editing/concepts) page doesn't work any more since our wiki is no longer operational.

**Comment by dju90** [comment]
Should we add any details on how .erb should not be used if the person wants their page to be translatable?

**Comment by Hamms** [comment]
Oh, good catch; I didn't audit any of the existing content, just moved it around. I'll see if I can track down the google docs version of that page, and remove that link if not.

I don't think it's necessary to add any stuff about translation, since that's still a manual process that requires developer intervention and advice. We can always add that in the future.

File: pegasus/sites.v3/code.org/public/editing/concepts.md.erb [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+	`---`<br>
+	`title: Your page title`<br>
+	`theme: responsive`<br>
+	`---`<br>
+
+	There are other kinds of metadata that you can include on your page:
+	* If you are on a page that has a few related pages, you'll want a side navigation menu that can be included with `nav: markdown_nav`. You can use a nav that already exists, otherwise you'll need an engineer to create a new nav.
+	* If you don't have a side nav but want a skinnier page layout, you can use both `nav: blank` and `rightbar: blank`
+	* Social media tags for Facebook and Twitter. [See how to here](/markdown/more).
```
Comment (breville) [inline pegasus/sites.v3/code.org/public/editing/concepts.md.erb]: maybe "how-to"?

File: pegasus/sites.v3/code.org/public/editing/markdown.md [hunk 2]
```diff
@@ -117,27 +133,6 @@ Links can be created by using brackets `[ ]` for the text of the link and parent
...
-### What your page will look like
-
-[<button>Sign up</button>](/professional-development-workshops)
-&nbsp;&nbsp;
-[<button>Join us</button>](https://hourofcode.com)
-
-
 ## Images
 ***
 You can include any image by first uploading it to Dropbox in the code.org/public/images folder. If you are uploading several images, try to create a folder inside images so you don't upload all of your images in the main images folder. Including an image is very similar to a link, except starting with an exclamation point at the beginning.
```
Comment (breville) [inline pegasus/sites.v3/code.org/public/editing/markdown.md]: "inside images" threw me at first.. maybe "inside `images/`"?

File: pegasus/sites.v3/code.org/public/editing/partials.md.erb [hunk 3]
```diff
@@ -0,0 +1,485 @@
...
+* (required) `id`: Can be any id name, just needs to be unique from all other videos on the same page. Try to name the id something related to the video content.
+* (required) `video_code`: This code is from YouTube. For example, if the URL for the video is https://www.youtube.com/watch?v=ip051U7Rvds then the video code is `ip051U7Rvds`
+* (optional) `caption`: Appears below the video thumbnail
+* (optional) `play_button`: Use `center` to appear in the middle of the thumbnail or `caption` to appear below the video.
+* (optional) `facebook` and `twitter`: 
+* (optional) `letterbox`: Set to `false` if you don't want the black bars in the thumbnail
+
+When YouTube is blocked by a school, we will use a different video player. In this event, you'll want to include both a download link and a new image thumbnail.
+
+* (required) `download_path`: If this is a video made by Code.org, include it's Amazon hosted download link. You can [ask an engineer](http://wiki.code.org/display/PROD/How+to+add+website+content+via+Dropbox+or+Gsheets#HowtoaddwebsitecontentviaDropboxorGsheets-Addingvideos) for the video link.
```
Comment (breville) [inline pegasus/sites.v3/code.org/public/editing/partials.md.erb]: nit: its Amazon

File: pegasus/sites.v3/code.org/public/editing/partials.md.erb [hunk 4]
```diff
@@ -0,0 +1,485 @@
...
+
+## Social Media
+***
+
+### How to point to Code.org's social media sites
+Do not use any social media components that are hosted on third-party sites. Third-party content can use cookies to track users, violating our privacy policy. Instead, use an image file that we host. This image may still link to the third-party site when clicked.
+
+<%= view :social_media, facebook: "Code.org", twitter: "codeorg", tumblr: "http://teacherblog.code.org", instagram: "codeorg" %>
+
+### How to share your page on social media
```
Comment (breville) [inline pegasus/sites.v3/code.org/public/editing/partials.md.erb:163]: I wonder if it's worth calling out our special centralised code for a few pages in particular?  https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/src/social_metadata.rb

**Review by breville — COMMENTED** [review]
This is fantastic living documentation.  Really nice work!

**Review by breville — APPROVED** [review]

File: pegasus/sites.v3/code.org/public/editing/partials.md.erb (hunk 4 above)
Comment (Hamms) [reply pegasus/sites.v3/code.org/public/editing/partials.md.erb:163]: That seems worth doing! I'll leave that for a separate change, though.

These files are all available for editing via dropbox, too

**Pull request merged**